### ISSUE TITLE
No module named 'samexporter'

### ISSUE THREAD

**Issue by morestart** [vietanhdev/samexporter#6]
```
python.exe: Error while finding module specification for 'samexporter.export_encoder' (ModuleNotFoundError: No module named 'samexporter')
python.exe: Error while finding module specification for 'samexporter.export_decoder' (ModuleNotFoundError: No module named 'samexporter')
```

**Comment by morestart** [comment]
I used inference.py to load segment_anything_vit_h_encoder_quant.onnx,but I received an error:

onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Invalid rank for input: x Got: 3 Expected: 4 Please fix either the inputs or the model.

**Comment by satyajandhyala** [comment]
__init__.py file is missing under samexporter subfolder. Create an empty file the it works fine.

**Comment by vietanhdev** [comment]
@satyajandhyala Could you create a pull request to fix it?
Thank you very much!

**Comment by Greg7000** [comment]
Please note this issue occurs with PyPi 0.2. Consider providing new release with this fix

**Comment by vietanhdev** [comment]
Published v0.3.0 for this issue. Thank @Greg7000 !

**Comment by GuangmingCao** [comment]
> I used inference.py to load segment_anything_vit_h_encoder_quant.onnx,but I received an error:
> 
> onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Invalid rank for input: x Got: 3 Expected: 4 Please fix either the inputs or the model.

have you solved it ?
